README.rst

```diff
@@ -1,9 +1,22 @@
-***************
 craft-platforms
 ***************
 
-Utilities for handling platform information
+.. image:: https://readthedocs.com/projects/canonical-craft-platforms/badge/?version=latest
+    :target: https://canonical-craft-platforms.readthedocs-hosted.com/en/latest/?badge=latest
```

Comment:
Curious that the current status is "unknown" 😆

Reply:
Oop, I meant to ask about making the documentation public!

Reply:
Fixed, now it works